Make code cloud agnostic (#23)
nginxinc/nginx-asg-sync: README.md

region: us-west-2
api_endpoint: http://127.0.0.1:8080/api
sync_interval_in_seconds: 5

> cloud_provider: "AWS"

Reviewer: For consistency with other string fields
```suggestion
cloud_provider: AWS
```

* The `region` key defines the AWS region where we deploy NGINX Plus and the Auto Scaling groups.
* The `api_endpoint` key defines the NGINX Plus API endpoint.
* The `sync_interval_in_seconds` key defines the synchronization interval: nginx-asg-sync checks for scaling updates every 5 seconds.

> * The `cloud_provider` key defines cloud provider that will be used. By default is `AWS`. This means, this field can be empty if using AWS.

Reviewer: ```suggestion
* The `cloud_provider` key defines a cloud provider that will be used. The default is `AWS`. This means the key can be empty if using AWS.
```
